docs/developer/eventing/sources/containersource/reference.md

```diff
@@ -1,5 +1,7 @@
 # ContainerSource reference
 
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
```

Comment:
the badge should be green (v1 API). `API_Version-v1-red` -> `API_Version-v1-green`

Reply:
Thanks. I'll update the remaining red badges.